src/decoration/decoration.ts

```diff
@@ -0,0 +1,33 @@
+import * as vscode from 'vscode';
+import * as path from 'path';
+import * as fs from 'fs';
+import { getNotes } from '../note-db';
+
+const decorationType = vscode.window.createTextEditorDecorationType({
+    backgroundColor: new vscode.ThemeColor('codeAnnotaion.mainColor')
+});
+
+export const setDecorations = (): void => {
+    const openEditor = vscode.window.visibleTextEditors[0];
+
+    const ranges: vscode.Range[] = [];
+    getNotes().forEach( note => {
+        if (note.fileName === openEditor.document.fileName){
```

Comment:
Please a space between ) and {:

```suggestion
        if (note.fileName === openEditor.document.fileName) {
```